doc/sphinx/compiling.rst

```diff
@@ -32,69 +33,267 @@ Installation Prerequisites
 Linux
 -----
 
-* For Ubuntu or Debian users, the following packages should be installed using
+General Notes
+^^^^^^^^^^^^^
+
+* To download the source code, installing ``git`` is highly recommended.
+
+* SCons is only available for Python 2, so building the Python 3 module requires
+  two installations of Python (one of Python 2 and one of Python 3), even if you
+  do not intend to build the Python 2 module.
+
+* The following instructions use the system-installed versions of Python, but
+  alternate installations such as the Anaconda distribution of Python can be
+  used as well.
+
+* Cython is only required to be installed for the version of Python that also
+  has SCons installed; following the instructions below will install Cython for
+  the version of Python 2 installed in the system directories.
+
+* The SUNDIALS suite of solvers used in Cantera is available from several sources.
+  The sources in most distribution repositories may be out of date, or not
+  optimized for your hardware. The source code for SUNDIALS will be downloaded
+  and compiled with the rest of Cantera, so installing from your distribution's
+  repositories is optional.
```

Comment:
Do we want users to prefer the system installation? I thought one of the reasons to include the source in the Cantera compile was to try and minimize issues linking to other sources for SUNDIALS. My thought with this was that this part of the docs should be a minimal set of dependencies just to get up and running, with options (like system SUNDIALS, fmt, Eigen, etc.) explained later.